### DISCUSSION TITLE
Vercel Deployment Problems

### DISCUSSION THREAD

**Discussion by franziska-caroline** [neuefische/questions#797] [Web] [answered]
Hello,
We are currently working on the Card component and we were trying to deploy our updates on vercel. Now were are getting an error message that we don't understand.

We have committed and pushed the new status on the branch feature-plant-list and created a PR [(https://github.com/MartaJaekel/Plantbuddy/pull/9)]. 

Description of the error:
We got the error message that some checks were not successful and that the status is canceled.

ERROR

![Bildschirmfoto 2023-12-12 um 12 46 21](https://github.com/neuefische/questions/assets/143591286/7aaff476-21ae-400a-9f71-188eabddeea6)

Search for the keywords to learn more about each error.
What have I already tried:
We already searched online for this error and I tried to redeploy, deploy on a different branch. However, this error keeps coming up and we don't know what the problem is.

We are happy about any help! 😇

**Comment by marcbendixen** [comment]
It looks like it's working now in your [new PR](https://github.com/MartaJaekel/Plantbuddy/pull/10)!?

**Reply by franziska-caroline** [reply, marked as answer]
Yes thank you! Jessica just helped us :)
